src/corefx/System.Globalization.Native/pal_calendarData.c

```diff
@@ -654,6 +620,7 @@ extern "C" int32_t GlobalizationNative_GetJapaneseEraStartDate(
                     if (U_FAILURE(err))
                         return false;
 
+                    ucal_close(pCal);
```

Comment:
Move it before the if? Also, there are couple of places with return above where the `ucal_close(pCal);` is missing.

Reply:
It was before if, not it has been moved down as part of https://github.com/dotnet/coreclr/commit/32addbf5fd4a30df9941fd139f5f7d6bdb41ab18